Content.Server/Atmos/GasMixture.cs

```diff
@@ -146,7 +145,7 @@ public void Archive()
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public void Merge(GasMixture giver)
+        public void Merge(GasMixture? giver)
         {
             if (Immutable || giver == null) return;
```

Comment:
Remove this `giver == null` then?

Reply:
This is adding nullability to the `giver` argument though so if I do that it's going to cause a compile error in 152 when accessing giver, or should I remove both the nullability and the null check?

Reply:
Yeah, `giver` should never be null so I guess removing both the check and nullability is what should be done here

Reply:
oh yeah I misread the diff, my bad.